src/__tests__/timerUtils.ts

```diff
@@ -1,11 +1,9 @@
-// @flow
-
 import { setTimeout } from '../helpers/timers';
 
 const TimerMode = {
   Legacy: 'legacy',
   Modern: 'modern', // broken for now
```

Comment:
Still broken? :)

Reply:
No idea to be honest, I'm not super familiar with this modern/legacy timer. But I guess it'd be worth investigating after this PR!